pkgs/development/compilers/emscripten/wrapped.nix

```diff
@@ -0,0 +1,84 @@
+# TODO ATN: this is missing a lot of features that cc-wrapper has, but cc-wrapper also has features that conflict with emscripten. What to do?
```

Comment:
Can you explain in a bit more detail what the conflicts are?

Reply:
I'm not sure anymore. I've rewritten this PR a little to wrap Emscripten in a way that can be `wrapCC`ed, and that seems to work better.